ansible/roles/g_nodes/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+default_usr: root
+work_dir: /usr/local
```

Comment:
Perhaps these variable names can be improved. In Ansible, variables are scoped a bit loosely, so it's better to have descriptive, unique names like `gputest_workdir_path` and `gputest_workdir_owner`.

Reply:
References to these variables need to be changed as well.